web/packages/teleport/src/Discover/Database/CreateDatabase/useCreateDatabase.ts

```diff
@@ -146,6 +156,18 @@ export function useCreateDatabase() {
         await ctx.databaseService.createDatabase(clusterId, db);
         setCreatedDb(db);
       } catch (err) {
+        // Check if the error is a result of an existing database.
+        if (err instanceof ApiError) {
+          if (
+            err.response.status === 409 ||
+            err.message.toLowerCase().includes('already exists')
+          ) {
+            return attemptDbServerQueryAndBuildErrMsg(
+              db.name,
+              Boolean(db.awsRds && db.awsRds.accountId)
```

Comment:
I think `Boolean()` here is redundant as you're using `&&`

Reply:
i get this type error if i don't cast it:
```
Argument of type 'string' is not assignable to parameter of type 'boolean'
```